18F/dol-whd-14c
# File Upload Validation

#### What's this PR do?
File upload will not allow larger than 5 MB and not supported file types through API calls.
The allowed file types are .docx, .pdf, .jpg, .jpeg, .png., .xls, .csv and .doc
Individual file upload limit: 5MB
#### Where should the reviewer start?
Create a few test files. The test files should include a large file that is more than 5 MB and a file that is not part of the supported file types.
#### How should this be manually tested?
Login to the application. Upload a large file (greater than 5 MB) into the application, the file should be ignored. Upload another file that is less than 5 MB and the file type is one of the supported file types; this file should be uploaded.
#### Any background context you want to provide?
No error message will return for invalided files at this time.
#### What are the relevant tickets?
#263
#### Screenshots (if appropriate)
#### Questions:
